main.go

```diff
@@ -261,7 +261,7 @@ func DoStashCPSingle(sourceFile string, destination string, methods []string, re
 		destination = dest_url.Path
 	}
 
-	if source_scheme_pieces[0] == "stash" || source_scheme_pieces[0]== "osdf" {
+	if source_scheme_pieces[0] == "stash" || source_scheme_pieces[0] == "osdf" {
```

Comment:
What happens when the URL is `tokenname+stash://` ?

Reply:
Turns out, I already had tests that caught the change.  `tokenname+stash://` is tested here: https://github.com/htcondor/osdf-client/blob/f50a5d16c4b7a4a8308fb4d3ebae4697ac9dd898/main_test.go#L126